src/App.js

```diff
@@ -17,8 +17,8 @@ function App() {
   const changeHandler = function(event) {
       const content = event.target.files[0]
       const url = URL.createObjectURL(content)
-
-      read_csv(url, {start: 0, end: 20}).then(df => {
+      
+      read_csv("https://raw.githubusercontent.com/plotly/datasets/master/finance-charts-apple.csv", {start: 0, end: 20}).then(df => {
```

Comment:
remove this link and add back the default `url`  as `read_csv(URL)` before pushing

Reply:
should it be capital URL?

Reply:
no the former variable before adding this

Reply:
ok

Reply:
have you resolve the error in the danfo js lib